pkg/ibmcsidriver/controller_helper.go

```diff
@@ -329,6 +334,20 @@ func getVolumeParameters(logger *zap.Logger, req *csi.CreateVolumeRequest, confi
 
 	//TODO port the code from VPC BLOCK to find region if zone is given
 
+	// validate bandwidth for dp2 profile
+	if volume.VPCVolume.Profile.Name == DP2Profile && volume.VPCVolume.Bandwidth > 0 {
+		err = fmt.Errorf("bandwidth must be specified for dp2 profile fileShare. provide a non-zero 'bandwidth' value in the storage class parameters")
```

Comment:
Don`t mention anything of bandwidth for dp2. Just say

`Bandwidth is not supported for dp2 profile. Please remove the property or set it to zero`